client/src/components/Lesson Page/section.js

```diff
@@ -11,6 +12,14 @@ const Section = (props) => {
   const [selectedAns, setSelectedAns] = useState(null)
   // Logs if chosen answer is correct
   const [isCorrect, setIsCorrect] = useState(null);
+  // Formats concepts to have paragraph breaks
+  const bodyWithLineBreaks = lesson.concept.replace(/\\n/g, '\n');
+  const paragraphs = bodyWithLineBreaks.split('\n');
+
+  // Formats text to not send images to text-to-audio API
+  const conceptToAudio = paragraphs.join('').replace(/<img[^>]*>/g);
+  const questionToAudio = lesson.question.replace(/<img[^>]*>/g);
+
```

Comment:
The need for all these replacements strikes me as a code smell; the backend should be smart enough to send the frontend what it needs.

That said, not a major issue.